### PR TITLE
Initialize the repository

### PR DESCRIPTION
**Pull request by johnkrovitch**

### CODE REVIEW

File: src/StringUtils.php [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+class StringUtils
+{
+    /**
+     * Camelize a string.
+     *
+     * @param string $id A string to camelize
+     *
+     * @return string The camelized string
+     */
+    public static function camelize($id): string
```
Comment (lutangar) [inline src/StringUtils.php]: There could be a flag to return a UpperCamelCase or lowerCamelCase

File: tests/StringUtilsTest.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    public function testCamelize(): void
+    {
+        $this->assertEquals('MyLittlePanda', StringUtils::camelize('my little panda'));
+        $this->assertEquals('MyLittlePanda', StringUtils::camelize('My Little Panda'));
+    }
+
+    public function testUnderscore(): void
+    {
+        $this->assertEquals('salsifi-potatoes', StringUtils::underscore('Salsifi-Potatoes'));
```
Comment (lutangar) [inline tests/StringUtilsTest.php:47]: The underscore function keep the dashes? `-`

**Review by lutangar — APPROVED** [review]

File: tests/StringUtilsTest.php (hunk 2 above)
Comment (johnkrovitch) [reply tests/StringUtilsTest.php:47]: I think it should preserve the dashes to preserve the same behavior as Symfony service.

File: src/StringUtils.php (hunk 1 above)
Comment (johnkrovitch) [reply src/StringUtils.php]: Done

**Pull request merged**